CONTRIBUTE.md

```diff
@@ -0,0 +1,45 @@
+## Setup panel
+
+1. clone the plugin to `%installationpath%/Grafana/grafana/plugins`
+2. Install dependencies `npm i`
+3. Build project `yarn build`
+
+## Setup demo env with minikube and (altered) sockshop
+
+### Minikube
+
+```
+minikube start --memory 8192 --cpus 4
```

Comment:
some people( like me ) would have a problem with the virtual machine and the minikube cluster cannot be correctly built with only these specifications . so sometimes the user have to add ' --no-vtx-check' to that line. you can add that as a small helper in case the user is stuck

Reply:
✅